integration/__tests__/command-palette.tests.ts

```diff
@@ -40,7 +40,7 @@ describe("Lens command palette", () => {
 
     it("opens command dialog from menu", async () => {
       await app.electron.ipcRenderer.send("test-menu-item-click", "View", "Command Palette...");
-      await app.client.waitUntilTextExists(".Select__option", "Preferences: Open");
+      await app.client.waitUntilTextExists(".Select__option", "Hotbar: Switch");
```

Comment:
Is this a better test?

Reply:
No, and I revert it back in https://github.com/lensapp/lens/pull/2779 because this PR removes that command dialog registration.